src/components/Input/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,31 @@
+import { cx } from '@emotion/css';
+import React, { PropsWithChildren } from 'react';
+import { TextAreaProps } from './TextAreaProps';
+import { textAreaStyle, stySizeDefault, styStatusDefault } from './styles';
+
+const TextArea = (props: PropsWithChildren<TextAreaProps>): JSX.Element => {
+    const { placeholder, text, txtAreaSize, txtAreaStatus } = props;
```

Comment:
you should actually put default value here
```suggestion
    const { placeholder, text, txtAreaSize = 20, txtAreaStatus = 'status' } = props;
```